docs/guides/api-tokens.md

```diff
@@ -0,0 +1,88 @@
+# Protecting an API with Access Tokens
+
+Access Tokens can be used to authenticate users for your own site, or when allowing third-party developers to access your API. When making requests using access tokens, the token should be included in the `Authorization` header as a `Bearer` token.
+
+To issue tokens for users, the `UserModel` must use the `CodeIgniter\Shield\Authentication\Traits\HasAccessTokens` trait. The `UserModel` that ships with Shield already uses this trait.
+
+```php
+use CodeIgniter\Shield\Authentication\Traits\HasAccessTokens
+use CodeIgniter\Model;
+
+class UserModel extneds Model
+{
+    use HasAccessTokens;
+}
+```
+
```

Comment:
We now require all User models to be an extension of ours, so this will always be the case. This can be dropped:
```suggestion
```